.github/workflows/scripts/biome_disable_check.py

```
@@ -0,0 +1,160 @@
+
+    try:
+        with open(file_path, encoding="utf-8") as file:
+            content = file.read()
+            return bool(biome_ignore_pattern.search(content))
+    except FileNotFoundError:
+        print(f"File not found: {file_path}")
+        return False
+    except PermissionError:
+        print(f"Permission denied: {file_path}")
+        return False
+    except (IOError, OSError) as e:
```

Ruff (0.11.9)

Replace aliased errors with `OSError`

Replace with builtin `OSError`

(UP024)